examples/aws/tidb-cluster.yaml

```diff
@@ -47,8 +55,8 @@ spec:
     service:
       annotations:
         service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
-        service.beta.kubernetes.io/aws-load-balancer-internal: '0.0.0.0/0'
-        service.beta.kubernetes.io/aws-load-balancer-type: nlb 
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
```

Comment:
Please do not set this to `internet-facing`, we use internal LB by default, if some users want to use internal-facing LB, they can just update this.
BTW, to keep backward compatibility, what about also adding:
`service.beta.kubernetes.io/aws-load-balancer-internal: "true"`?